docs/httplug.md

```diff
@@ -5,22 +5,36 @@ Httplug is an abstraction for HTTP clients. There are two main use cases:
 1. Usage in a project
 2. Usage in a reusable package
 
-In both cases, the client provides a `sendRequest` method to send a PSR-7 `RequestInterface` and returns a PSR-7 `ResponseInterface` or throws an exception that implements `Http\Client\Exception`.
+In both cases, the client provides a `sendRequest` method to send a PSR-7 `RequestInterface` and returns a PSR-7 `ResponseInterface` 
+or throws an exception that implements `Http\Client\Exception`.
+
+It can also provides a `sendAsyncRequest` method to send an asynchronous PSR-7 `RequestInterface` and returns a `Http\Client\Promise`, 
```

Comment:
lets say: There is also the HttpAsyncClient which provides the sendAsycRequest method to send a request asynchronously and...